backend/models/cans.py

```diff
@@ -150,6 +151,14 @@ class Agreement(BaseModel):
 
     notes = Column(Text, nullable=True)
 
+    @property
+    def status(self):
+        subq = select(
```

Comment:
We may change this. We can keep it for now, but we will want to review this before utilizing it in the frontend or for any other business logic calculations.